apps/wakunode2/wakunode2.nim

```diff
@@ -410,16 +410,9 @@ proc setupProtocols(node: WakuNode, conf: WakuNodeConf,
 
       try:
         await node.mountRlnRelay(rlnConf)
-      except:
+      except CatchableError:
         return err("failed to mount waku RLN relay protocol: " & getCurrentExceptionMsg())
 
-  if conf.swap:
```

Comment:
should we also remove the flag?
https://github.com/waku-org/nwaku/blob/master/apps/wakunode2/config.nim#L261